### PR TITLE
Add -v global flag for verbosity

### LINKED ISSUE #959: tsuru-client: add a verbose mode
Add a parameter "-v" to shows what http calls the client are doing.

### PR DESCRIPTION
**Pull request by msabramo**
Verbosity: 1 => print HTTP requests; 2 => print HTTP requests/responses

Fixes #959 

E.g.:

```
❯ tsuru -v 2 app-info -a someapp
*************************** <Request uri="/apps/someapp"> **********************************
GET /apps/someapp HTTP/1.1
Host: tsuru.corp.surveymonkey.com:8080
Connection: close
Authorization: bearer 6efcfbba38127d5f3c9a7121a8833e76ccb7f5cc

*************************** </Request uri="/apps/someapp"> **********************************
*************************** <Response uri="/apps/someapp"> **********************************
HTTP/1.1 404 Not Found
Connection: close
Transfer-Encoding: chunked
Content-Type: text/plain; charset=utf-8
Date: Sat, 27 Dec 2014 00:50:34 GMT
Supported-Crane: 0.6.0
Supported-Tsuru: 0.14.0
Supported-Tsuru-Admin: 0.8.1

17
App someapp not found.

0

*************************** </Response uri="/apps/someapp"> **********************************
Error: App someapp not found.
```

### CODE REVIEW

File: cmd/client.go [hunk 1]
```diff
@@ -51,11 +53,29 @@ func (c *Client) Do(request *http.Request) (*http.Response, error) {
 		request.Header.Set("Authorization", "bearer "+token)
 	}
 	request.Close = true
+	if c.Verbosity >= 1 {
+		fmt.Printf("*************************** <Request uri=%q> **********************************\n", request.URL.RequestURI())
```
Comment (msabramo) [inline cmd/client.go]: Cool, thanks!. Changed it to use `c.Context.Stdout` in the latest version.

**Comment by msabramo** [comment]
Added tests. Good to go now?

**Comment by cezarsa** [comment]
Thanks!

**Comment by scorphus** [comment]
Thanks, @msabramo!

**Pull request merged**